website/docs/examples/multi_client_geoimage/README.md

```diff
@@ -0,0 +1,236 @@
+ the `stamp_diff` variable. This threshold depends on the precision of your application.
+
+For educational purposes, we left a lot of `print` statements in the code to
+visualize the flow of the program. We recommend to remove them later in your application.
+
+```python
+async def _subscribe(self, subscription: SubscribeRequest) -> None:
+  # the client name is the last part of the query
+  client_name: str = subscription.uri.query.split("=")[-1]
+  client: EventClient = self.clients[client_name]
+  # subscribe to the event
+  async for event, message in client.subscribe(subscription, decode=True):
+      print(f"Received event from {client_name}{event.uri.path}")
+      if isinstance(message, oak_pb2.OakFrame):
+          await self.image_queue.put((event, message))
+      elif isinstance(message, gps_pb2.GpsFrame):
+          stamp_gps = get_stamp_by_semantics_and_clock_type(
+              event, semantics="service/send", clock_type="monotonic"
+          )
+          if stamp_gps is None:
+              continue
+
+          geo_image = None
+
+          while self.image_queue.qsize() > 0:
+              event_image, image = await self.image_queue.get()
+              stamp_image = get_stamp_by_semantics_and_clock_type(
+                  event_image, semantics="service/send", clock_type="monotonic"
+              )
+              if stamp_image is None:
+                  continue
+              stamp_diff = abs(stamp_gps - stamp_image)
+
+              # NOTE: define this threshold depending on the precision of your application
+              if stamp_diff > 0.05:
+                  print(f"Skipping image because stamp_diff is too large: {stamp_diff}")
+                  continue
+              else:
+                  print(f"Synced image and gps data with stamp_diff: {stamp_diff}")
+                  geo_image = ((event_image, image), (event, message))
+                  break
+
+          if geo_image is None:
+              print("Could not sync image and gps data")
+              continue
+```
+
+In addition, we provide a `config.json` file example that contains the configuration of the
+service. Notice that the subscribers are against the `oak0` and `gps` services,
+in particular to the `/rgb` and `/pvt` paths respectively.
+
+```json
+{
+  "configs": [
+    {
+      "name": "gps",
+      "port": 3001,
+      "host": "localhost"
+    },
+    {
+      "name": "oak0",
+      "port": 50010,
+      "host": "localhost"
+    },
+    {
+      "name": "multi_subscriber",
+      "subscriptions": [
+        {
+          "uri": {
+            "path": "/rgb",
+            "query": "service_name=oak0"
+          },
+          "every_n": 2
+        },
+        {
+          "uri": {
+            "path": "/pvt",
+            "query": "service_name=gps"
+          },
+          "every_n": 1
+        }
+      ]
+    }
+  ]
+}
+```
+
+## 4. Run the client
+
+In a terminal, run the client:
+
+```bash
+python main.py --config config.json
```

Comment:
If you do end up changing args to `--service-config` as in other examples, update here accordingly